File: answer.py
def two_sum(nums, t):
    for i, x in enumerate(nums):
        for j, y in enumerate(nums):
            if i != j and x + y == t:
                return [i, j]

def two_sum(numbers, target):
  list_index=[]
  for x in range(len(numbers)):
    for y in range(len(numbers)):
      if x != y and numbers[x]+numbers[y]==target:
            list_index.extend([x,y])
  return list(dict.fromkeys(list_index))

File: answer.js
const twoSum = (numbers,target) => {
  let arrResult = []
  numbers.forEach((x,i,arr)=>arr.forEach((y,j)=>i!=j&&x+y==target?arrResult.push(i,j):arrResult))
  return arrResult.slice(0,2)
}

function twoSum(numbers, target) {
    for (var i = 0; i < numbers.length-1; i++) {
        for (var j = i+1; j < numbers.length; j++) {
            if (numbers[i] + numbers[j] === target) return [i, j];
        }
    }
}

function twoSum(numbers, target) {
  let seen = new Map();
  for (let i = 0; i < numbers.length; i++) {
    let x = numbers[i], y = target - x;
    if (seen.has(y))
      return [seen.get(y), i];
    seen.set(x, i);
  }
}
